components/NavBar.tsx

```diff
@@ -0,0 +1,47 @@
+import { AppBar, Toolbar, Typography, Button, makeStyles } from '@material-ui/core'
+import React, { useContext } from 'react'
+import NextLink from 'next/link'
+
+import ElevationScroll from './ElevationScroll'
+import { AppContext } from './AppContext'
+
+interface Props {
+	title: string
+}
+
+const useStyles = makeStyles({
+	title: {
+		flexGrow: 1,
+	},
+})
+
+export default function NavBar(props: Props): JSX.Element {
+	const { themeType, setThemeType } = useContext(AppContext)
+	const classes = useStyles()
+
+	const handleThemeInvert = (): void => {
+		if (setThemeType) {
+			setThemeType(themeType === 'dark' ? 'light' : 'dark')
+		}
+	}
+
+	return (
+		<ElevationScroll>
+			<AppBar position="sticky">
+				<Toolbar>
+					<NextLink href="/">
+						<Typography variant="h5" className={classes.title}>
+							{props.title}
+						</Typography>
```

Comment:
`Typography` should be replaced or wrapped with a `MuiLink` here, unless you've done it this way for a reason?

Reply:
The MUI Link component has some CSS for visited links, and since the AppBar examples all use typography.

Reply:
Good thinking, in that case probably just need a `component="a"`.